src/components/marker-chart/Canvas.js

```diff
@@ -301,11 +301,13 @@ class MarkerChartCanvasImpl extends React.PureComponent<Props> {
       if (w2 > textMeasurement.minWidth) {
         const fittedText = textMeasurement.getFittedText(text, w2);
         if (fittedText) {
-          ctx.fillStyle = 'white';
+          ctx.fillStyle = 'black';
```

Comment:
Hmm, the commit message says "Change colors to black on light", but it's changing it to black when it's highlighted too. Was that intentional?

Reply:
I think I've been a bit too fast on this, I'll double check the colors for the highlighted case